pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -492,7 +497,34 @@ func (pl *dynamicResources) isSchedulableAfterClaimChange(logger klog.Logger, po
 		return framework.QueueSkip, nil
 	}
 
-	logger.V(4).Info("status of claim for pod got updated", "pod", klog.KObj(pod), "claim", klog.KObj(modifiedClaim))
+	logger.V(5).Info("status of claim for pod got updated", "pod", klog.KObj(pod), "claim", klog.KObj(modifiedClaim))
+	return framework.Queue, nil
+}
+
+// isSchedulableAfterPodChange is invoked for update pod events reported by
+// an informer. It checks whether that change adds the ResourceClaim(s) that the
+// pod has been waiting for.
+func (pl *dynamicResources) isSchedulableAfterPodChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
Don't we need any tests for that?

Reply:
hey, @macsko @pohly I am working on this PR, which seems to be related to this. If necessary, I can add this part of the test case to this PR :)
https://github.com/kubernetes/kubernetes/pull/127523

Reply:
@macsko: yes, this needs tests. Is it okay if @googs1025 adds them as a follow-up?

Reply:
Sure, can be this way

Reply:
This is not how we do things. And even more so when we have gotten rid of "test freeze", meaning that code needs to merge with all appropriate coverage.

I've let similar situations slide in the past, but IIRC, DRA is targeting beta now, so we can't be as forgiving as we were in previous iterations of DRA.

Reply:
@alculquicondor +1, we need to be sure we merge code and tests together from now on

Reply:
Fair enough. I've added unit tests.